packages/kbn-sharedux-components/src/empty_state/no_data_views/no_data_views.tsx

```diff
@@ -7,15 +7,13 @@
  */
 
 import React, { useCallback, useEffect, useRef } from 'react';
-
-import { DataView } from '../../../../../data_views/public';
-import { useEditors, usePermissions } from '../../../services';
-import type { SharedUXEditorsService } from '../../../services/editors';
+import { useEditors, usePermissions } from '@kbn/sharedux-services';
+import type { SharedUXEditorsService } from '@kbn/sharedux-services';
 
 import { NoDataViews as NoDataViewsComponent } from './no_data_views.component';
 
 export interface Props {
-  onDataViewCreated: (dataView: DataView) => void;
+  onDataViewCreated: (dataView: unknown) => void;
```

Comment:
I want there to be a better type for this.  I've asked in the Typescript group.

Reply:
What's wrong with `DataView`?

Reply:
oh, to avoid import

Reply:
@majagrubic DataView is a type from `src/plugins/data_views/public`.  Packages cannot depend on plugins.